### DISCUSSION TITLE
Configure target branch for onboarding PR other than the master

### DISCUSSION THREAD

**Discussion by nchery** [renovatebot/renovate#20051] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.112.1

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

5.9.0

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Basically I have a master branch BUT it is used to merge only released version. So i'd like to configure renovate so it targets a 'develop' branch as target branch for the onboarding PR
By default it takes as target branch the master branch

### What you tried so far.

I tried to use the variable [OnBoardingBranch](https://docs.renovatebot.com/self-hosted-configuration/#onboardingbranch) but somehow what it did was to take the branch defined in that variable to be merged in the Master branch rather than using it as target.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Config must be in the default branch. Within that you configure which baseBranches to target update PRs

**Reply by nchery** [reply]
ok so if I can't change the base branch for the onboarding PR (unless i change the default branch or i change the release process), the only choice i have is to skip the onboarding PR process ?

**Reply by nchery** [reply]
I've set the RENOVATE_ONBOARDING to false on the baseBranch renovate.json, however it keeps creating the onboarding PR. I've seen this [thread](https://github.com/renovatebot/renovate/discussions/8812) which was answered some months back. Does it require to have a renovate.json on the Default branch, and if not it creates the onboarding PR despite the RENOVATE_ONBOARDING=false ?

**Reply by rarkins** [reply]
`onboarding` is a global config option, not a repo config option. i.e., if you run your own bot, you must put it in the env before you run the bot.

You can have a nearly _empty_ config in the defaults branch, just with `baseBranches` and `useBaseBranch` config configured. Then the rest can be in the base branch.
